go.mod

```diff
@@ -13,7 +13,9 @@ require (
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
 	golang.org/x/sys v0.0.0-20201221093633-bc327ba9c2f0 // indirect
+	google.golang.org/api v0.36.0
 	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/apimachinery v0.18.8
```

Comment:
Let's avoid adding k8s.io dependencies as much as possible, especially since we need it only seem to be adding this for using sets.

Reply:
Done.